package capkproto;

import "capk_globals.proto";

message order_cancel_replace {
    optional bytes		orig_order_id = 1;
    optional bytes		cl_order_id = 2;
	optional bytes		strategy_id = 3;
    optional sfixed32	handl_inst = 4;
    optional string		symbol = 5;
	optional side_t		side = 6;
	optional double		order_qty = 7;
	optional double		price = 8;
	optional string		transact_time = 9;
	optional sfixed32		ord_type = 10;
	optional sfixed32		time_in_force = 11;
}

